modules/cudf/src/series/numeric.ts

```diff
@@ -1407,6 +1407,82 @@ export abstract class NumericSeries<T extends Numeric> extends Series<T> {
     return this._process_reduction(skipna, memoryResource)._col.var(ddof, memoryResource);
   }
 
+  /**
+   * Return Fisher’s unbiased kurtosis of a sample.
+   * Kurtosis obtained using Fisher’s definition of kurtosis (kurtosis of normal == 0.0). Normalized
+   * by N-1.
+   *
+   * @param skipna Exclude NA/null values. If an entire row/column is NA, the result will be NA.
+   * @param memoryResource The optional MemoryResource used to allocate the result Series's device
+   *   memory.
+   * @returns The unbiased kurtosis of all the values in this Series.
+   * @example
+   * ```typescript
+   * import {Series} from '@rapidsai/cudf';
+   * const a = Series.new([1, 2, 3, 4]);
+   *
+   * a.kurtosis() // -1.1999999999999904
+   * ```
+   */
+  kurtosis(skipna = true, memoryResource?: MemoryResource) {
+    if (this.length == 0 || (this.hasNulls && !skipna)) { return NaN; }
+
+    const data = this._process_reduction(skipna, memoryResource);
+
+    const n = data.length;
+    if (n < 4) { return NaN; }
+
+    const V = data.var(skipna, 1, memoryResource);  // ddof = 1
+    if (V == 0) { return 0; }
+
+    const mu = data.mean(skipna, memoryResource);
+
+    const m4 =
+      Number(data.sub(mu, memoryResource).pow(4, memoryResource).sum(skipna, memoryResource)) /
```

Comment:
After similarly using the `Number(...)` constructor in my https://github.com/rapidsai/node-rapids/pull/220 PR, Paul pointed out that there would be information loss if we were to convert a `bigint` to a `number`. 

Is this a concern in `kurtosis` and `skew`? I know `mean()` returns a number, so unless you upcast (if you even can), you wouldn't be able to do the division on line 1441.

Reply:
I'm not sure about upcasting. When the values are actually big, then the results seem reasonable (tho different) either way:
```
> BigInt(1000000000000000000000000000000000000000000000000) / BigInt(3)
333333333333333347948614348358732451544682550613n

> Number(BigInt(1000000000000000000000000000000000000000000000000)) / 3
3.3333333333333336e+47
```

However if you are dealing with "BigInt" values that are actually not, in fact, very big for whatever reason, then the upcasting results do not seem OK at all:
```
> Number(BigInt(10)) / 3
3.3333333333333335
> BigInt(10) /  BigInt(3)
3n
```

Definitely would like to hear from @trxcllnt in case there's just some other option I'm not aware of

Reply:
handled per discussion w/ @trxcllnt